flink-ml-servable-core/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.flink</groupId>
+    <artifactId>flink-ml-parent</artifactId>
+    <version>2.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>flink-ml-servable-core</artifactId>
+  <name>Flink ML : Servable : Core</name>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.apache.flink</groupId>
```

Comment:
It seems that `flink-ml-servable-core` still relies on `flink-core` and is not consistent with the goal of FLIP-289.
Can you explain why do we move XXTypeInfo into `flink-ml-servable-core`? 

Moreover, there are some classes that do not depend on flink runtime, but not used in `flink-ml-servable-core` (e.g., `MLMetrics`). Can we move it back where it is used?

Reply:
@lindong28 and I had a discussion about `flink-core` in a previous draft PR [1]. We agreed this change can be put in a follow-up PR.

[1] https://github.com/apache/flink-ml/pull/199#discussion_r1090122715

Reply:
Thanks for the explanation. I agree it is a reasonable solution in the long term.